db/migrate/20151212191508_add_unique_index_to_course_levels.rb

```diff
@@ -0,0 +1,8 @@
+class AddUniqueIndexToCourseLevels < ActiveRecord::Migration
+  def change
+    add_index :course_levels,
+              [:experience_points_threshold, :course_id],
+              unique: true,
+              name: 'add_unique_index_to_course_levels'
```

Comment:
then what I do is I take the generated name and trim the end part. it's usually a few repeated column names or table names that I just factor out.

the name should not have any verbs, should be purely a noun.